php-frontend/src/main/java/org/sonar/plugins/php/api/visitors/TreeVisitor.java

```diff
@@ -19,16 +19,44 @@
  */
 package org.sonar.plugins.php.api.visitors;
 
-import com.google.common.annotations.Beta;
-import org.sonar.php.tree.impl.expression.IdentifierTreeImpl;
+import org.sonar.php.tree.impl.expression.ParenthesizedExpressionTreeImpl;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'org.sonar.php.tree.impl.expression.ParenthesizedExpressionTreeImpl'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AUselessImportCheck)